joomla/joomla-cms
# Added missing space in article layout

Pull Request for Issue # .

### Summary of Changes
Adding missing space in the com_content article layout when using the menu item page class.

### Testing Instructions
Create a single article and a menu item linked to that article.
Apply a "Page Class" in the created menu item in the tab "Page Display".
Make sure there is no extra space before the class name.
Visit the frontend and take a look at the menu class that is added to the body class and the article layout class.

![Image001](https://user-images.githubusercontent.com/5610413/174747641-61a65f28-d1ce-4807-93a1-c728d45f93d3.jpg)

### Actual result BEFORE applying this Pull Request
There is no space in the article layout class.
See screenshot

### Expected result AFTER applying this Pull Request
There should be a space in the article layout class.
See screenshot

### Documentation Changes Required
No
